cs3api4lab/__init__.py

```diff
@@ -1,19 +1,35 @@
-from notebook.utils import url_path_join
+import json
+from pathlib import Path
 
+from .handlers import setup_handlers
 from ._version import __version__
-from cs3api4lab.api.cs3apismanager import CS3APIsManager
```

Comment:
Already asked above. By removing this then we need to have a not as clean configuration. Not a big issue, but I'm curious to know why we had to remove it.